Tools-Override/resolveContract.targets

```diff
@@ -30,7 +31,9 @@
     <ItemGroup Condition="'@(ContractProject)' != ''">
       <!--Don't flow the values for Configuration --> 
       <ContractProject> 
-        <UndefineProperties>Configuration;%(ContractProject.UndefineProperties)</UndefineProperties> 
+        <UndefineProperties Condition="'$(HasBuildConfigurations)' != 'true' and '%(ContractProject.TargetGroup)' == ''">Configuration;%(ContractProject.UndefineProperties)</UndefineProperties>
```

Comment:
I've merged so when you fix the conflicts you should be able to undo any changes to this file.